infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -252,8 +252,8 @@ public void unableToRetrieveValidatorStatusesFromBeaconNode() {
     log.error("Unable to retrieve validator statuses from BeaconNode.");
   }
 
-  public void validatorStatus(String validatorStatus, String publicKey) {
-    log.info("Validator {} status is {}.", validatorStatus, publicKey);
+  public void validatorStatus(String publicKey, String validatorStatus) {
+    log.info("Validator {} status is {}.", publicKey, validatorStatus);
```

Comment:
Shouldn't this be?:
```suggestion
  public void validatorStatus(String validatorStatus, String publicKey) {
    log.info("Validator {} status is {}.", publicKey, validatorStatus);
```

Reply:
Now that I've changed the public key to come first on the other side, this makes sense. Should I change both of them?

Reply:
Oh, sorry - I was viewing a filtered changeset, so it looked like we only changed it here.